.github/workflows/periodic.yml

```diff
@@ -17,6 +17,10 @@ jobs:
           kubectl_version: v1.29.2
           version: v0.22.0
           node_image: kindest/node:v1.29.2
+      - name: Build images
+        run: ./scripts/deploy/github/build-images.sh
+      - name: Deploy kfp-tekton
+        run: ./scripts/deploy/github/deploy-kfp.sh
```

Comment:
These new lines are supposed to be only used in kfp-tekton backend tests. Periodic tests don't run any tests against kfp-tekton.

Reply:
This one I don't really fully get! We need to define which components of the kfp we actually need in the kind cluster to be able to run the tests. The prow tests run against the cluster that is specified [here](https://github.com/kubeflow/testing/tree/master/test-infra/kfp).

If I try to run GH actions tests against bare kind cluster or against [standalone KFP](https://www.kubeflow.org/docs/components/pipelines/v1/installation/standalone-deployment/), I get a missing endpoint error: 
<img width="958" alt="image" src="https://github.com/kubeflow/pipelines/assets/116455436/c682ff75-09ed-4f03-927b-f08d75034d14">

As far as I understand, that means that Client class cannot connect to kfp server. 

A possible solution in this case is that I actually *need* to specify the host address. Could you give a possible hint what the kfp host address available in GH actions can be? I think I already tried:
http://ml-pipeline:8888/
http://ml-pipeline-ui:80

Reply:
I guess we can run the tests against standalone kfp then? I realised that my tests didn't work since I didn't allow for any time for the kfp pods to come up in the test cluster.